maintainers/maintainer-list.nix

```diff
@@ -1202,6 +1202,12 @@
     github = "beardhatcode";
     githubId = 662538;
   };
+  beezow = {
```

Comment:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME``.